.babelrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
Created common babelrc which will be imported in each workspace. babelrc is required in each workspace to run jest tests